new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/WebLogicImageTool.java

```diff
@@ -124,4 +142,35 @@ private String buildList(List<String> list) {
     }
     return strList;
   }
+  
+  /**
+   * Add WDT installer to the WebLogic Image Tool cache
+   * @return true if the command succeeds 
+   */
+  public boolean addInstaller() throws FileNotFoundException {
+    try {
+      checkFile(WDT_ZIP_PATH);
+    } catch (FileNotFoundException fnfe) {
+      logger.warning("Failed to create an image due to Exception: " + fnfe.getMessage());
+      throw fnfe;
+    }
+    return executeAndVerify(
```

Comment:
much better to build the string first, then return it - easier to debug

Reply:
Done